section1/practice_exercises/ex3.rb

```diff
@@ -0,0 +1,37 @@
+# This line prints the string
+puts "I will now count my chickens:"
+
+# This line counts the hens and interpolates a mathematical expression
+puts "Hens #{25.0 + 30.0 / 6.0}"
+# This line counts the Roosters and interpolates a mathematical expression
+puts "Roosters #{100.0 - 25.0 * 3.0 % 4.0}"
+
+# This line prints the string
+puts "Now I will count the eggs:"
+
+# This line computes and prints a mathematical expression
+puts 3.0 + 2.0 + 1.0 - 5.0 + 4.0 % 2.0 - 1.0 / 4.0 + 6.0
```

Comment:
Challenge yourself: Why does this line print out "6.75" when using floats verus "7" when using integers?